public/sass/components/_slate_editor.scss

```diff
@@ -95,45 +95,45 @@
     color: $text-color-weak;
   }
 
-  .token.punctuation {
-    color: $text-color-weak;
+  .token.variable,
+  .token.entity {
+    color: $text-color;
```

Comment:
Metric names are tokenized as `variable`. With this change you can no longer tell if a metric name exists on the platform, because now it's just the text color. Before, it was a different color.

Reply:
ah, did not realize that it validated the metric name exists, and it then gets a different color. thought it was only based on the syntax parser. 

Tried to find a good color, but found the readability best with a white text for this part. If everything has colors it just becomes too much. But think a red color could work for this, or maybe invert it so it's red if it's not found, so default to that.